library/pool.c

```diff
@@ -1184,46 +1197,129 @@ int dnet_io_init(struct dnet_node *n, struct dnet_config *cfg)
 	return err;
 }
 
+int dnet_server_backend_init(struct dnet_node *n, size_t backend_id)
```

Comment:
dnet_schedule_io() called from network thread. It is very undesirable to slowdown it, especially in case of tens of thousands RPS.